js/dom.js

```diff
@@ -1,7 +1,13 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-unused-vars */
+/* eslint-disable func-names */
+/* eslint-disable no-undef */
+/* eslint-disable no-loop-func */
```

Comment:
- [ ] [OPTIONAL] Right now there's a lot of `disable linter` lines, remember that Eslint rules are meant to catch bad code and enforce best practices. 
![image](https://user-images.githubusercontent.com/73254040/99016493-da7eff00-251c-11eb-9b6a-a1ce00808f03.png)